src/clients/c++/library/CMakeLists.txt

```diff
@@ -86,9 +89,12 @@ add_library(
   TRTIS::request_static ALIAS request_static
 )
 if(${TRTIS_ENABLE_GPU})
-  target_include_directories(
+  target_include_directories(request_static PRIVATE ${CUDA_INCLUDE_DIRS})
+  target_link_libraries(
     request_static
-    PRIVATE ${CUDA_INCLUDE_DIRS}
+    PRIVATE -L/usr/local/cuda/lib64/stubs
```

Comment:
Do we need stubs and nvidia-ml (I believe that is only needed for nvml metrics), or is just CUDA_LIBRARIES sufficient?

Reply:
You are right. CUDA_LIBRARIES should be enough